examples/hosts/iframe-host/src/outer.ts

```diff
@@ -2,35 +2,25 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  * Licensed under the MIT License.
  */
+
+import { fluidExport as TodoContainer } from "@fluid-example/todo";
+import { IFluidCodeDetails } from "@fluidframework/container-definitions";
+import { Container, Loader } from "@fluidframework/container-loader";
+import { IFluidObject } from "@fluidframework/core-interfaces";
+import { InsecureTinyliciousUrlResolver } from "@fluidframework/get-tinylicious-container";
 import {
     RouterliciousDocumentServiceFactory,
 } from "@fluidframework/routerlicious-driver";
-import { IFluidCodeDetails } from "@fluidframework/container-definitions";
-import { BaseHost } from "@fluidframework/base-host";
-import { IRequest } from "@fluidframework/core-interfaces";
-import { InsecureUrlResolver } from "@fluidframework/test-runtime-utils";
-import { SemVerCdnCodeResolver } from "@fluidframework/web-code-loader";
+// import { InsecureUrlResolver } from "@fluidframework/test-runtime-utils";
 import { HTMLViewAdapter } from "@fluidframework/view-adapters";
 import { IFrameOuterHost } from "./inframehost";
 
-const createRequest = (): IRequest => ({
-    url: `${window.location.origin}/testdoc50`,
-});
-
-const getTinyliciousResolver =
-    () => new InsecureUrlResolver(
-        "http://localhost:3000",
-        "http://localhost:3000",
-        "http://localhost:3000",
-        "tinylicious",
-        "12345",
-        { id: "userid0" },
-        "bearer");
+const documentId = `testdoc${Math.floor(Math.random() * 10000)}`;
```

Comment:
The random here makes it so the doc is lost when refreshing the page and could theoretically have name collisions (although rare) - consider doing something like in the app-integration samples using a timestamp or other predictably-unique ID, plus putting it in the page URL so page refresh works.

Reply:
Playing with it a bit, looks like this will expose some bugs in the refresh scenario that also need ironed out.

Reply:
there's the issue with how outer doesn't handle the refresh that i need to cover with the host refactoring, were there any other issues that you saw?

Reply:
I didn't look super hard, just made the small change to see if it would magically work or not :)  I think that's probably what I was seeing though, I think the inner was working correctly if I remember right